Microsoft.Toolkit.Uwp.UI.Controls/Graph/ProfileCard/ProfileCard.cs

```diff
@@ -0,0 +1,104 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.IO;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Graph
+{
+    /// <summary>
+    /// The Profile Card control is a simple way to display a user in multiple different formats and mixes of name/image/e-mail.
+    /// </summary>
+    public partial class ProfileCard : Control
+    {
+        private static readonly BitmapImage PersonPhoto = new BitmapImage(new Uri("ms-appx:///Microsoft.Toolkit.Uwp.UI.Controls/Graph/Assets/person.png"));
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ProfileCard"/> class.
+        /// </summary>
+        public ProfileCard()
+        {
+            DefaultStyleKey = typeof(ProfileCard);
+        }
+
+        /// <summary>
+        /// Override default OnApplyTemplate to initialize child controls
+        /// </summary>
+        protected override void OnApplyTemplate()
+        {
+            base.OnApplyTemplate();
+            InitUserProfile();
+        }
+
+        private static void OnGraphAccessTokenChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
```

Comment:
These changed methods should go in properties file.

Reply:
@michael-hawker Moved to Events file as per other feedbacks for consistence.